src/com/aca/homework/week2/random/string/RandomString.java

```diff
@@ -0,0 +1,21 @@
+package com.aca.homework.week2.random.string;
+
+import java.util.Random;
+
+public class RandomString {
+    public static void main(String[] args) {
+        for (int i = 0; i <10 ; i++) {
+            returnRandomString();
+        }
+    }
+    public static void returnRandomString(){
+        String [] array= {"a","b","w","7","8","9","u"};
+        String newArray = "";
```

Comment:
the variable name is not good. it is an ordinary string so there is no need to name it newArray.